scripts/us_census/acs5yr/subject_tables/common/generate_col_map_test.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for generateColMap."""
+import os
+import json
+import unittest
+
+from .generate_col_map import generate_stat_var_map
+
+
+class GenerateColMapTest(unittest.TestCase):
+    """ Test Cases for checking the generation of column map from JSON Spec"""
+
+    def test_generating_column_map(self):
+        """tests the column map on a subset of columns from S2702 """
+        spec_path = './testdata/spec.json'
+        column_file = './testdata/columns.txt'
+        expected_out = './testdata/expected.json'
+
+        # base path from where the script runs
+        base_path = os.path.dirname(__file__)
+
+        # load the files
+        f = open(os.path.join(base_path, spec_path), 'r')
+        spec_dict = json.load(f)
+        f.close()
```

Comment:
may be useful to have a function that does this as it is repeated multiple time in this file.